Control coverage/Feature/CloudService.md

```diff
@@ -2,4 +2,5 @@
\ No newline at end of file
+| <b>ControlId:</b><br>Azure_CloudService_SI_Enable_AntiMalware<br><b>DisplayName:</b><br>Antimalware extension must be installed on cloud service roles. <br><b>Description: </b><br> Enable the Antimalware extension for the cloud service roles.| <b> ARM API to get Cloud Service extension details: </b> <br> /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/<br>Microsoft.ClassicCompute/domainNames/{cloudServiceName}/slots/{slotName}/<br>roles/{roleName}/extensionReferences?api-version=2015-06-01  <br><b>Properties:</b><br> name: 'PaaSAntimalware-****' <br> properties.state| <b>Passed: </b><br> Antimalware extension is enabled for all the roles in this cloud service. <br><b>Failed: </b><br> Antimalware extension is not enabled for one or more roles in this cloud service. |
+| <b>ControlId:</b><br>Azure_CloudService_SI_Disable_RemoteDesktop_Access<br><b>DisplayName:</b><br>Disable Remote Desktop (RDP) access on cloud service roles<br><b>Description: </b><br>Disable Remote Desktop (RDP) access on cloud service roles. | <b>ARM API to get the list of Deployment Slots in a Cloud Service:</b><br> /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/<br>Microsoft.ClassicCompute/domainNames/{cloudServiceName}/<br>deploymentSlots?api-version=2016-11-01<br><b>Properties:</b><br>properties.slotType, properties.configuration<br><b>ARM API to get the list of Cloud Service Roles in a Deployment Slot:</b><br> /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/<br>Microsoft.ClassicCompute/domainNames/{cloudServiceName}/<br>slots/{slotName}/roles?api-version=2016-04-01<br><b>Properties:</b><br>[\*].name<br><b>ARM API to get the list of Extensions in a Cloud Service Role:</b><br> /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/<br>Microsoft.ClassicCompute/domainNames/{cloudServiceName}/<br>slots/{slotName}/roles/{roleName}/extensionReferences?api-version=2015-06-01<br><b>Properties:</b><br>[\*].properties.name, [\*].properties.state | <b>Passed:</b><br>Remote Desktop access is not enabled for any Cloud Service Role.<br><b>Failed:</b><br>Remote Desktop access is enabled for Cloud Service Role(s). |
```

Comment:
I would say, while breaking the lines in APIs, new line should starts with '/', like below:
/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers
/Microsoft.ClassicCompute/domainNames/{cloudServiceName}
/slots/{slotName}/roles?api-version=2016-04-01

Just a suggestion, take your call.

Reply:
If a line is not terminated with a `/` or `?`, it can cause anyone to assume that the query was ending there. Hence, terminating all intermediate lines with `/` or `?`